veros/diagnostics/overturning.py

```diff
@@ -110,6 +141,13 @@ def diagnose(self, vs):
         sig_loc_face = 0.5 * (sig_loc[2:-2, 2:-2, :] + sig_loc[2:-2, 3:-1, :])
         trans = np.zeros((vs.ny + 4, self.nlevel), dtype=vs.default_float_type)
         z_sig = np.zeros((vs.ny + 4, self.nlevel), dtype=vs.default_float_type)
+
+        # Potential density using TEOS-10
+        self.prho[2:-2, 2:-2, :] += density.gsw.gsw_pot_rho_t_exact(vs,
```

Comment:
What if we're using one of the other EOS formulations?

Reply:
I am not aware of any other EOS formulations. Are there any different polynomial methods developed and approved by IOC for potential density calculation? 
Or I misunderstand you...?

Reply:
In Veros we support different formulations of the equation of state, e.g. this one: https://github.com/dionhaefner/veros/blob/master/veros/core/density/nonlinear_eq1.py

Does your formulation make sense to use even if we're not using GSW in the density routines? How should that case be handled?

Reply:
It is not completely wrong, but Markus said that the potential density for other EOS will be the same as in-situ density. 
Let me reiterate my commits. Is it better to make an extra function for the potential density computation in veros/veros/core/density/get_rho.py or put if statement (EOS=5 then prho computation) in the overturning diagnostic?